client/web/src/insights/components/insights-view-grid/components/insight-card/components/insight-card-menu/InsightCardMenu.tsx

```diff
@@ -9,7 +9,7 @@ import { positionRight } from '../../../../../context-menu/utils'
 import styles from './InsightCardMenu.module.scss'
 
 export interface InsightCardMenuProps {
-    className?: string
+    menuButtonClasses?: string
```

Comment:
For consistency, I would probably call this property `menuButtonClassName` (singular)
```suggestion
    menuButtonClassName?: string
```

Reply:
Great idea, will do :+1: